packages/sakuli-legacy/src/context/sahi/assertion/assertion-api.function.spec.ts

```diff
@@ -134,10 +134,18 @@ describe("assertion-api", () => {
       expect(_containsTextMock).toHaveBeenCalledWith(dummyQuery, "ABC");
     });
 
-    it("should throw an exception, if element does not contains text", async () => {
+    it("should throw an exception with default message, if element does not contains text", async () => {
+      //GIVEN
       _containsTextMock.mockResolvedValue(false);
-      await expect(api._assertContainsText("ABC", dummyQuery)).rejects.toThrow(
-        AssertionError
+      const expectedText = "ABC";
+
+      //WHEN + THEN
+      await expect(
+        api._assertContainsText(expectedText, dummyQuery)
+      ).rejects.toThrow(
+        new AssertionError({
+          message: `The element does not contain the expected text "${expectedText}" but "${dummyQuery.identifier}"`,
+        })
```

Comment:
I'm missing the test scenarios of _"should throw configured exception message"_, to ensure that a user provided exception message is used instead of the default.

Reply:
Luckily those were already implemented :smile:

Reply:
nice :+1: